assets/src/application/ui/forms/espressoForm/styles.scss

```diff
@@ -75,16 +75,17 @@
 			&-pair {
 				width: 50%;
 				display: inline-flex;
+				padding-right: var(--ee-margin-default);
+				margin: var(--ee-margin-default) 0;
+				label {
+					display: flex;
+					align-items: center;
+				}
 			}
-		}
-		.form-item-pair {
-			width: 50%;
-			display: inline-flex;
-			padding-right: var(--ee-margin-default);
-			margin: var(--ee-margin-default) 0;
-			label {
-				display: flex;
-				align-items: center;
+
+			&__help-info {
+				color: var(--ee-color-primary-low-contrast);
```

Comment:
IMO, it's better to centralize color, to make it more consistent with other icons styles;
when doing previous refactor I noticed that is a bit hard to maintain these kinds of local patched styles

Reply:
hmmm  ya this is case is different than the others since it is on the coloured background of the sidebar, but you're right that that could be encoded somehow maybe with an `inverse` prop or something

Reply:
yes, `inverse` prop sounds like an interesting approach